src/auto/output_stream.rs

```diff
@@ -128,6 +128,26 @@ pub trait OutputStreamExt: 'static {
         bytes: &glib::Bytes,
         io_priority: glib::Priority,
     ) -> Box_<dyn future::Future<Output = Result<isize, Error>> + std::marker::Unpin>;
+
+    //#[cfg(any(feature = "v2_60", feature = "dox"))]
+    //fn writev<P: IsA<Cancellable>>(&self, vectors: /*Ignored*/&[&OutputVector], cancellable: Option<&P>) -> Result<usize, Error>;
```

Comment:
recod `GOutputVector` contains only fields and need manual implementation.

Reply:
Yes, and ideally with zero-cost conversion from `std::io::IoVec` :)